Apps/US/DynamicsGPMigrationVendor1099/app/src/Upgrade/GPVendor1099MigUpgrade.Codeunit.al

```diff
@@ -0,0 +1,14 @@
+codeunit 42001 "GP Vendor 1099 Mig. Upgrade"
```

Comment:
I assume we do not need this codeunit, right?

Reply:
This will contain upgrade logic when new IRS 1099 form/box changes need to be supported by this extension.